EcpBtBot/MainClass.py

```diff
@@ -26,5 +26,14 @@ def reponseBot(self, message):
         user_message = message
 
         rep = Response(self.ai._get_json_response(user_message))
+        QuickReplies=[]
```

Comment:
use either camel case (quickReplies) or snake case (quick_replies), but starting with uppercase is usually kept for class names only